pkg/cmd/config/init/config_init.go

```diff
@@ -214,8 +215,8 @@ func cloneRepositories(
 	cplatformRepoFullname git.RepositoryFullname,
 	templatesRepoFullname git.RepositoryFullname,
 ) (cloneRepositoriesResult, error) {
-	cloneReposSpinner := streams.Spinner("Cloning repositories...")
-	defer cloneReposSpinner.Done()
+	streams.Wizard("Cloning repositories", "Cloned repositories")
```

Comment:
can we include which repositories, it will help understand what is hapenning

Reply:
The repositories are printed out in the Info messages a bit further down